lib/StoryEngine/DataLayer/model/ChoiceDataModel.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+
+namespace StoryEngine.StoryEngineDataModel;
+
+public record ChoiceDataModel(OutcomeDataModel Outcome)
```

Comment:
This is strictly the outcome for a choice right? Not the initial prompt?

Reply:
At least one choice must be given for each story node.  If there is only one choice given, it means there is no choice involved, and the associated outcome will be directly applied.  Otherwise, the outcome associated with whatever choice the user picks will be applied.